target/bin/open-dkim

```diff
@@ -2,6 +2,8 @@
 
 # shellcheck source=../scripts/helpers/index.sh
 source /usr/local/bin/helpers/index.sh
+# shellcheck source=/dev/null
+source /etc/dms-settings
```

Comment:
Does that work without DMS running? If you use `setup.sh` to setup DKIM it skips the startup scripts doesn't it?

Reply:
It does indeed _not_ work haha